daemon/k8s_watcher.go

```diff
@@ -1908,7 +1910,7 @@ func (d *Daemon) updateCiliumNetworkPolicyV2(ciliumV2Store cache.Store,
 		logfields.K8sNamespace:                     newRuleCpy.ObjectMeta.Namespace,
 		"annotations.old":                          oldRuleCpy.ObjectMeta.Annotations,
 		"annotations":                              newRuleCpy.ObjectMeta.Annotations,
-	}).Debug("Modified CiliumNetworkPolicy")
```

Comment:
I know the why of this change but consider that this will be extremely verbose, I think this should be kept as Debug. Otherwise one can argue, why aren't all "modifies" Infos?

Reply:
Sure, that's fine. This is still wip though, I will mark it as pending-review when it is ready.